packages/migrations/src/actions/2023.01.18T11.03.41.registry-v2.sql

```diff
@@ -0,0 +1,117 @@
+-- 
+
+-- Describes an action that was performed on a registry.
+-- PUSH - a new schema was pushed to the registry, an existing schema was updated
+-- DELETE - a schema was deleted from the registry
+CREATE TYPE schema_registry_action AS ENUM ('PUSH', 'DELETE');
```

Comment:
note that enum can not be altered - no new members can be easilz added - so it might be better to make this a text column with insert check or keep the enum validation on the application side

Reply:
Yeah, we used it in other places as well. In other projects (lance) we had this problem multiple times :/